### ISSUE TITLE
Applying fromLatLngToPoint and then fromPointToLatLng doesn't return the original value

### ISSUE THREAD

**Issue by R2D221** [mapsplugin/cordova-plugin-googlemaps#1016]
I'm trying to do some calculations on the position of the camera, but was getting very frustrated because it didn't seem to be working correctly. Turns out there's a problem with the conversions

Take the following coordinates:

    var latLng = new plugin.google.maps.LatLng(19.393669,-99.1767865);

If you run fromLatLngToPoint and fromPointToLatLng the following way:

    map.fromLatLngToPoint(latLng, function (coords) {
    	map.fromPointToLatLng(coords, function (latLng) {
    		console.log(latLng);
    	});
    });

You get different coordinates:

    [19.39452329701747, -99.17627893388271]
    // Sidenote: why is this an array and not a LatLng?

In the actual map, the problem is quite noticeable, sending the camera a block away than where it was supposed to be.

Tested this using the latest version in master (I uninstalled it and reinstalled it to be sure it wasn't already fixed).

My device is a Samsung Galaxy J5 (SM-J500M) running Android 5.1.1, Android System WebView 51.0.2704.81, Maps 9.27.2.

**Comment by wf9a5m75** [comment]
Ask to Google.
Code is here.
https://github.com/mapsplugin/cordova-plugin-googlemaps/blob/master/src/android/plugin/google/maps/PluginMap.java#L578-L604

**Comment by R2D221** [comment]
Sorry, it wasn't clear to me. Are you saying that a bug experienced in your plugin is of no concern to you if the bug is in one of your dependencies?

**Comment by wf9a5m75** [comment]
This plugin just converting points <--> LatLng.
If this is a critical bug, please post with the correct code.

**Comment by R2D221** [comment]
I used a workaround not using pixels at all, but that's not the point of me reporting this bug. The point is that it can get fixed so that other people who need to use this function don't get greeted with unexpected behavior.

I understand what you say that the bug may be with Google themselves, but since your plugin is called “Google Maps plugin for Cordova”, shouldn't you at least be worried if Google is messing up?

I'm exclusively developing in Cordova, so I have no way to test if a base Android project shows the same buggy behavior, but I thought that reporting this would help you (or somebody else, since this is open source, right?) go into the right direction into fixing it.

Anyway, this is all the info I got. If you're not interested in further investigating the issue, then that's a shame for this project, because it has been useful to me so far.
